readme.md

```diff
@@ -1,33 +1,35 @@
+This repo is also meant to facilitate better feedback among programmers. An editorial process like code review will help programmers write cleaner code. Engaging in code review exposes programmers to other styles and helps with better understanding likes and dislikes about different coding styles for finding your own.
+
+While the focus of this content is on Python, there are fundmental similarities between it and R that are critical to understand for any programmer. These fundamentals will be covered as an exercise in identifying the utility of these comparable languages, and why or where one may be preferably used over another. 
 
 ## Using This Repo:
 
-Although there is no explicit ordering to this content, when facilitating a workshop or working through these samples on your own I suggest the following order:
+Although there is no explicit ordering to this content, when facilitating a workshop or working through these samples on your own, the following order is suggested:
 
 1. [General Advice](general-advice.md)
-1. [Giving and Getting Feedback](giving-and-getting-feedback.md)
+1. [Giving and Receiving Feedback](giving-and-getting-feedback.md)
 1. [Examples](examples/)
     1. [Naming Variables](examples/variable-naming.py)
     1. [Brevity vs Clarity](examples/brevity-vs-clarity.py)
     1. [Don't Repeat Yourself](examples/DRY.py)
     1. [Documentation and Comments](examples/documentation-comments.py)
     1. [Code Structure](examples/structure.py)
-1. [Helpful Tools For Clean Code and Review](helpful-tools.md)
+1. [Helpful Tools and Resources For Clean Code and Review](helpful-tools.md)
 
 ## Copyright and Licensing
 
-The purpose of this repository is purely educational. The information and code here is provided without warranty of any kind. The text and code this repo has been released to the public domain with the exception of a piece of code copied from the Keras library found in [examples/documentation-comments.py](examples/documentation-comments.py). That Keras code is licensed under the MIT license, which is included in the code file itself as well as the LICENSE file. 
+The purpose of this repository is purely educational. The information and code here is provided without warranty. The text and code this repo has been released to the public domain with the exception of a piece of code copied from the Keras library found in [examples/documentation-comments.py](examples/documentation-comments.py). That Keras code is licensed under the MIT license, which is included in the code file itself as well as the LICENSE file. 
 
-The rest of the code and text is in the public domain and you may use it for any purpose whatsoever, without restriction.
+The rest of the code and text is in the public domain (CC0) and you may use it for any purpose, without restriction.
```

Comment:
The Unlicense text in the actual license file is not the same as the CC0 text, so we shouldn't include this.